source/administration/changelog.md

```diff
@@ -2,6 +2,168 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
+## Release v3.4.0
+
+Expected release date: 2016-09-16
+
+### Highlights
+
+#### Heroku Deployment
```

Comment:
Should this go further down since it's a preview? And Zapier integration to the top?